File: src/Entity/Blog.php
<?php

namespace App\Entity;

use App\Repository\BlogRepository;
use Doctrine\Common\Collections\ArrayCollection;
use Doctrine\Common\Collections\Collection;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;
use Symfony\Component\HttpFoundation\File\File;

/**
 * @ORM\Entity(repositoryClass="App\Repository\BlogRepository")
 * @ORM\Table(name="blog")
 * @ORM\HasLifecycleCallbacks()
 */
class Blog
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\NotBlank(message="Ce champ est obligatoire")
     * @Assert\Email(
     *     message = "The email '{{ value }}' is not a valid email."
     * )
     */
    private $email;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\NotBlank(message="Ce champ est obligatoire")
     * @Assert\Length(
     *      min = 2,
     *      max = 250,
     *      minMessage = "Vote description doit avoir {{ limit }} characters au minimum",
     *      maxMessage = "Votre description ne peut pas avoir plus que {{ limit }} characters"
     * )
     */
    private $description;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $pays;



    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\NotBlank(message="Ce champ est obligatoire")
     */
    private $titre;

    /**
     * @ORM\Column(name="photo", type="string", length=500)
     */
    private $photo;

    /**
     * @ORM\OneToMany(targetEntity=Commentaire::class, mappedBy="blog",cascade={"remove"}, orphanRemoval=true)
     */
    private $commentaires;

    /**
     * @ORM\Column(type="boolean")
     */
    private $etat;

    /**
     * @param $id
     * @param $email
     * @param $description
     * @param $pays
     * @param $titre
     * @param $photo
     * @param $commentaires
     * @param $etat
     */


    public function getId(): ?int
    {
        return $this->id;
    }

    public function getEmail(): ?string
    {
        return $this->email;
    }

    public function setEmail(string $email): self
    {
        $this->email = $email;

        return $this;
    }

    public function getDescription(): ?string
    {
        return $this->description;
    }

    public function setDescription(string $description): self
    {
        $this->description = $description;

        return $this;
    }

    public function getPays(): ?string
    {
        return $this->pays;
    }

    public function setPays(string $pays): self
    {
        $this->pays = $pays;

        return $this;
    }

    /**
     * @return mixed
     */
    public function getPhoto()
    {
        return $this->photo;
    }

    /**
     * @param mixed $photo
     */
    public function setPhoto($photo)
    {
        $this->photo = $photo;
    }

    public function getTitre(): ?string
    {
        return $this->titre;
    }

    public function setTitre(string $titre): self
    {
        $this->titre = $titre;

        return $this;
    }


    public function addCommentaire(Commentaire $commentaire): self
    {
        if (!$this->commentaires->contains($commentaire)) {
            $this->commentaires[] = $commentaire;
            $commentaire->setBlog($this);
        }

        return $this;
    }

    /**
     * @return mixed
     */
    public function getCommentaires()
    {
        return $this->commentaires;
    }

    /**
     * @param mixed $Commentaires
     */
    public function setCommentaires($Commentaires)
    {
        $this->commentaires = $Commentaires;
    }

    public function getEtat(): ?int
    {
        return $this->etat;
    }

    public function setEtat(int $etat): self
    {
        $this->etat = $etat;

        return $this;
    }


}

File: src/entities/Blog.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;

import com.mysql.jdbc.Blob;
import javafx.scene.image.Image;

/**
 *
 * @author houda
 */
public class Blog {
      private int id;
     private String email,description,pays,titre;
     private String photo;

    

   

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getPhoto() {
        return photo;
    }

    public void setPhoto(String photo) {
        this.photo = photo;
    }

   
    

  

    public String getTitre() {
        return titre;
    }

    public void setTitre(String titre) {
        this.titre = titre;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getPays() {
        return pays;
    }

    public void setPays(String pays) {
        this.pays = pays;
    }



    public Blog(String email, String description, String pays, String titre,  String photo) {
        this.email = email;
        this.description = description;
        this.pays = pays;
        this.titre = titre;
        this.photo=photo;
    }
        public Blog(int id,String email, String description, String pays, String titre) {
        this.id=id;
        this.email = email;
        this.description = description;
        this.pays = pays;
        this.titre = titre;
       
    }

        public Blog(String email, String description, String pays, String titre) {
        this.email = email;
        this.description = description;
        this.pays = pays;
        this.titre = titre;
       
    }
    public Blog(int id, String email, String description, String pays, String titre, String photo) {
        this.id = id;
        this.email = email;
        this.description = description;
        this.pays = pays;
        this.titre = titre;
        this.photo = photo;
    }

    public Blog() {
    }

    @Override
    public String toString() {
        return "Blog{" + "id=" + id + ", email=" + email + ", description=" + description + ", pays=" + pays + ", titre=" + titre + ", photo=" + photo + '}';
    }

    

    
    
     
    
}
